frontend/src/features/properties/selector/MapSelectorContainer.test.tsx

```diff
@@ -58,63 +54,60 @@ describe('MapSelectorContainer component', () => {
 
   beforeEach(() => {
     mockAxios
-      .onGet(
-        new RegExp(
-          'https://openmaps.gov.bc.ca/geo/pub/WHSE_CADASTRE.PMBC_PARCEL_FABRIC_POLY_SVW/wfs',
-        ),
-      )
-      .reply(200, mockPropertyLayerSearchResponse)
-      .onGet(
-        new RegExp(
-          'https://openmaps.gov.bc.ca/geo/pub/WHSE_ADMIN_BOUNDARIES.TADM_MOT_REGIONAL_BNDRY_POLY/wfs',
-        ),
-      )
-      .reply(200, mockMotiRegionLayerResponse)
-      .onGet(
-        new RegExp(
-          'https://openmaps.gov.bc.ca/geo/pub/WHSE_ADMIN_BOUNDARIES.TADM_MOT_DISTRICT_BNDRY_POLY/wfs',
-        ),
-      )
-      .reply(200, mockDistrictLayerResponse);
+      .onGet(new RegExp('ogs-internal/ows'))
+      .reply(200, mockFAParcelLayerResponse)
+      .onGet(new RegExp('tools/geocoder/nearest*'))
+      .reply(200, mockGeocoderOptions[0]);
   });
 
   afterEach(() => {
     jest.resetAllMocks();
   });
 
-  it('renders as expected when provided no properties', () => {
+  it('renders as expected when provided no properties', async () => {
     const { asFragment } = setup({});
-    expect(asFragment()).toMatchSnapshot();
+    await act(async () => {
+      expect(asFragment()).toMatchSnapshot();
+    });
   });
-  it('displays two tabs', () => {
+
+  it('displays two tabs', async () => {
     const { getByText } = setup({});
-    expect(getByText('Locate on Map')).toBeVisible();
+    await act(async () => {
+      expect(getByText('Locate on Map')).toBeVisible();
+    });
   });
-  it('displays locate on Map by default', () => {
+
+  it('displays locate on Map by default', async () => {
     const { getByText } = setup({});
-    expect(getByText('Locate on Map')).toHaveClass('active');
+    await act(async () => {
```

Comment:
is this just warning clean up?

Reply:
Yep, these tests where really throwing a fit about the state being changed